include/roboteam_ai/new_skills/SetDribbler.h

```diff
@@ -0,0 +1,26 @@
+//
+// Created by timovdk on 2/11/20.
+//
+
+#ifndef RTT_SETDRIBBLER_H
+#define RTT_SETDRIBBLER_H
+
+#include <include/roboteam_ai/skills/Skill.h>
+
+namespace rtt::ai {
+
+/**
+ * This skill sets the dribbler at a speed specified in the blackboard
+ */
+
+class SetDribbler : public Skill {
```

Comment:
I highly suggest if we want to make new skills to also make a new base class or at least refactor it.  Your code is mostly fine, but this base class is really ugly.

Reply:
I haven't seen it yet, ill check that out in a sec

Reply:
I will ask Jesse tomorrow, I was thinking the same